bolt12-tip-jar/src/main/scala/fr/acinq/eclair/plugins/tipjar/TipJarPlugin.scala

```diff
@@ -38,23 +45,53 @@ class TipJarPlugin extends Plugin with RouteProvider with Logging {
   }
 
   override def onSetup(setup: Setup): Unit = {
+    val intermediateNodes = if (setup.config.hasPath("tip-jar.intermediate-nodes")) {
+      setup.config.getStringList("tip-jar.intermediate-nodes").asScala.toSeq.map(s => PublicKey(ByteVector.fromValidHex(s)))
+    }
+    else {
+      Nil
+    }
```

Comment:
nit: the scala formatter puts `else` on the same line as the previous `{` (also applies to the other condition below):

```suggestion
    } else {
      Nil
    }
```